docs/install_guides/installing-model-dev-tools.md

```diff
@@ -8,17 +8,24 @@ If you want to download, convert, optimize and tune pre-trained deep learning mo
 * Post-Training Optimization Tool
 * Model Downloader and other Open Model Zoo tools
 
-> **NOTE**: From the 2022.1 release, the OpenVINO™ Development Tools can only be installed via PyPI. 
+The instructions on this page show how to install OpenVINO Development Tools. If you are a Python developer, it only takes a few simple steps to install the tools with PyPI. If you are developing in C++, OpenVINO Runtime must be installed separately before installing OpenVINO Development Tools.
 
-## For Python Developers
+In both cases, Python 3.6 - 3.9 need be installed on your machine before starting.
 
-If you are a Python developer, you can find the main steps below to install OpenVINO Development Tools. For more details, see <https://pypi.org/project/openvino-dev>.
+> **NOTE**: From the 2022.1 release, the OpenVINO™ Development Tools can only be installed via PyPI.
+
+## <a name="python-developers"></a>For Python Developers
+
+If you are a Python developer, follow the steps in the <a href="#install-dev-tools">Installing OpenVINO Development Tools</a> section on this page to install it. Installing OpenVINO Development Tools will also install OpenVINO Runtime as a dependency, so you don’t need to install OpenVINO Runtime separately. This option is recommended for new users.
+
+## <a name="cpp-developers"></a>For C++ Developers
+If you are a C++ developer, you must first install OpenVINO Runtime separately to set up the C++ libraries, sample code, and dependencies for building applications with OpenVINO. These files are not included with the PyPI distribution. See the [Install OpenVINO Runtime](./installing-openvino-runtime.md) page to install OpenVINO Runtime from an archive file for your operating system.
 
 While installing OpenVINO Development Tools, OpenVINO Runtime will also be installed as a dependency, so you don't need to install OpenVINO Runtime separately.
```

Comment:
This is not correct. We should copy what's in the PR for master.

Reply:
Has been resolved, see the following suggestions having been applied.

Reply:
Done in 
https://github.com/openvinotoolkit/openvino/pull/13868/commits/54336da4ae4cde00af1dc2c96d4f3d4e6b946821
https://github.com/openvinotoolkit/openvino/pull/13868/commits/2614c5468002bb940536fb037fceda3272df0d3e
https://github.com/openvinotoolkit/openvino/pull/13868/commits/49528ece751c985e5d998ab6e03e441b1f67e13e
https://github.com/openvinotoolkit/openvino/pull/13868/commits/915b7116abeb38d053a3bd1313fe603c4e5da6a3
https://github.com/openvinotoolkit/openvino/pull/13868/commits/6272678c61cb8e9e3fd3a2a98089b289b1e0fa14
https://github.com/openvinotoolkit/openvino/pull/13868/commits/deeb504f7c6f653c23c41113cb3cc2ac47cd7504

Reply:
This is even more confusing....where is "For C++ developers"? Can we apply the changes more carefully?